src/components/Form/index.tsx

```diff
@@ -0,0 +1,79 @@
+import Form from '@rjsf/mui';
+import ajvValidator from '@rjsf/validator-ajv6';
+import { FormProps as RjsFormProps } from '@rjsf/core';
+import { Box, BoxProps, Button, ButtonProps } from '@mui/material';
+import { PasswordWidget } from './PasswordWidget';
+import { SelectWidget } from './SelectWidget';
+import { Fragment } from 'react';
+
+const internalWidgets: {
+	[k: string]: any;
+} = {
+	PasswordWidget,
+	SelectWidget,
+};
+
+export interface RJSFormProps
+	extends Omit<RjsFormProps, 'validator' | 'onFocus' | 'onBlur'> {
+	/** If true, do not display the form submit button */
+	hideSubmitButton?: boolean;
+	/** Properties that are passed to the submit button, these are the same props used for the [`Button`](#button) component */
+	submitButtonProps?: ButtonProps;
+	/** If passed, it will show a secondary button, these are the same props used for the [`Button`](#button) component */
+	actionButtons?: ButtonProps[];
+
+	validator?: RjsFormProps['validator'];
```

Comment:
I don't see us using this property in the UI, can we remove this?

Reply:
Or is the point solely to make this prop optional? If so then maybe the following as a nit? Up to you:
```suggestion
	extends Omit<RjsFormProps, 'validator' | 'onFocus' | 'onBlur'> & Partial<Pick<RjsFormProps, 'validator'>>
```